vgscollect/src/main/java/com/verygoodsecurity/vgscollect/widget/CardVerificationCodeEditText.kt

```diff
@@ -43,11 +44,18 @@ class CardVerificationCodeEditText @JvmOverloads constructor(
                     getBoolean(R.styleable.CardVerificationCodeEditText_singleLine, true)
                 val scrollHorizontally =
                     getBoolean(R.styleable.CardVerificationCodeEditText_scrollHorizontally, true)
-                val gravity = getInt(R.styleable.CardVerificationCodeEditText_gravity, Gravity.START or Gravity.CENTER_VERTICAL)
+                val gravity = getInt(
+                    R.styleable.CardVerificationCodeEditText_gravity,
+                    Gravity.START or Gravity.CENTER_VERTICAL
```

Comment:
The default should have gravity END. Or it doesn't have an influence on the icon gravity in CVC field?

Reply:
@Endorf No this code does not have an impact on icons